README.md

```diff
@@ -18,6 +18,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [Hosting and Domain Conference India](http://www.hdcon.org/) | 3 Feb | Hyderabad | Hosting and Domain Conference is the World's Top initiative to discuss on hosting, domain, blogging and web technologies. |
 | [ReactFoo Pune](https://reactfoo.in/) | 19-20 Jan | Pune | A conference on the lessons learned while implementing React and React Native. |
 | [IN/Clojure](http://inclojure.org/) | 12-13 Jan | Bengaluru | IN/Clojure is India's annual Clojure conference. It is also the only Clojure conference in all of Asia.
+| [Gnunify 2018](http://gnunify.in/) | 16-17 Feb | SICSR, Pune | GNU/Linux and open source technologies and related. |
```

Comment:
This list is in the reverse chronological order. Can you please add this conference between RubyConf and FOSSMeet? Thanks!